### ISSUE TITLE
Image Component LongDescription

### ISSUE THREAD

**Issue by photodow** [carbon-design-system/ibm-dotcom-library#1417]
## Detailed description

For the template simplifcation team we forgot to add in a long description for more detailed descriptions of an image for accessibillity screen readers. I've updated the functional specs. We just need to add the capability to the Image component.

#### Additional Information
* [Functional specs](https://ibm.box.com/s/1bbrspqkipecv2o010l8czzese084cix)

#### Acceptance Criteria

- [ ] new longDescrption prop
- [ ] Screen reader can read it
- [ ] Not visible non screen reader users
